queries.sql

```diff
@@ -40,3 +40,61 @@ SELECT *
 FROM animals
 WHERE weight>=10.4 and weight<=17.3 
 
+/**ex-2*/
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon'
+
+/*Update the animals table by setting the species column to pokemon for all animals that don't have species already set*/
+UPDATE animals
+SET species="pokemon"
+WHERE species IS NULL;
+
+/* delete all records in the animals table, then roll back the transaction.*/
+DELETE FROM animals
+
+/* Delete all animals born after Jan 1st, 2022.*/
+DELETE FROM animals
+WHERE date_of_birth > "01/01/2022"
+
+/*Update all animals' weight to be their weight multiplied by -1*/
+UPDATE animals
+SET weight=weight*(-1)
+
+/*Update all animals' weights that are negative to be their weight multiplied by -1*/
+UPDATE animals
+SET weight=weight*(-1)
+WHERE weight < 0 ;
+
+/*How many animals are there?*/
+SELECT COUNT(id)
+FROM animals;
+
+/*How many animals have never tried to escape*/
+SELECT COUNT (escape_attempts)
+FROM animals
+WHERE escape_attempts="1"
```

Comment:
- [ ] The `escape_attempts` should be equal to zero to show the animals that have never try to escape